src/EOS/Name.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "../BinaryCoding.h"
+#include "../Data.h"
+
+using Data = TW::Data;
+
+namespace TW::EOS {
+class Name {
```

Comment:
`Name` is a bit general, how about `AccountName`?

Reply:
Not only account name but also all names on EOSIO system (e.g. account name, action name) should follow this convention.